src/ontology/edits/oeo-physical.omn

```diff
@@ -2134,11 +2134,14 @@ Class: OEO_00000299
 Class: OEO_00000300
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Nuclear energy is the use of nuclear reactions that release nuclear energy to generate heat, which most frequently is then used in steam turbines to produce electricity in a nuclear power plant. Nuclear power can be obtained from nuclear fission, nuclear decay and nuclear fusion. Presently, the vast majority of electricity from nuclear power is produced by nuclear fission of elements in the actinide series of the periodic table. Nuclear decay processes are used in niche applications such as radioisotope thermoelectric generators. The possibility of generating electricity from nuclear fusion is still at a research phase with no commercial applications. This article mostly deals with nuclear fission power for electricity generation."@en,
-        <http://purl.obolibrary.org/obo/IAO_0000119> "https://en.wikipedia.org/w/index.php?title=Nuclear_power&oldid=868476098"@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Nuclear binding energy is the energy that is required to disassemble the nucleus of an atom into its component parts."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000119> "Derived from https://en.wikipedia.org/wiki/Nuclear_binding_energy"@en,
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000119> "Derived from https://en.wikipedia.org/w/index.php?title=Nuclear_binding_energy&oldid=1007327561"@en,
```

I suggest using here a permalink to the current Wikipedia article version as Wikipedia articles sometimes change quite substantially over time.